app/controllers/comments_controller.rb

```diff
@@ -29,12 +40,23 @@ def set_comment
     @comment = Comment.find_by(id: params[:id])
   end
 
+  def create_comment
+    @create_comment ||= CreateComment.call(comment_params: comment_params,
+                                           current_user: current_user)
+  end
+
+  def update_comment
+    @update_comment ||= UpdateComment.call(comment_params: comment_params,
+                                           comment: @comment, current_user: current_user)
+  end
+
   def comment_params
     params.require(:comment)
       .permit(:content, :task_id)
       .merge(
         {
-          user: current_user
+          user: current_user,
+          task: @task
```

Comment:
At the same time you accept `task_id` and infer `task`. Could you remove some of them?